infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/LogReplicationMetadataManager.java

```diff
@@ -97,25 +100,81 @@ public LogReplicationMetadataManager(CorfuRuntime runtime, LogReplicationContext
         this.runtime = runtime;
         this.corfuStore = new CorfuStore(runtime);
         this.replicationContext = replicationContext;
+        this.metadataTable = (Table<LogReplicationSession, ReplicationMetadata, Message>)
+                tryOpenTable(this.corfuStore, NAMESPACE, METADATA_TABLE_NAME,
+                        LogReplicationSession.class, ReplicationMetadata.class, null);
+        this.statusTable = (Table<LogReplicationSession, ReplicationStatus, Message>)
+                tryOpenTable(this.corfuStore, NAMESPACE, LogReplicationUtils.REPLICATION_STATUS_TABLE_NAME,
```

Comment:
The name of this table got changed to ReplicationStatusSource in lr_v2.  As we had discussed earlier, can you change it back to the older name?  The new name is defined here -
https://github.com/CorfuDB/CorfuDB/blob/log_replication_v2/runtime/src/main/java/org/corfudb/runtime/LogReplicationUtils.java#L40

Reply:
Thanks so much for reminding! Done!